core/src/main/java/nu/yona/server/subscriptions/service/UserService.java

```diff
@@ -169,8 +170,8 @@ private void addMandatoryGoals(User userEntity)
 
 	private void addGoal(User userEntity, ActivityCategoryDTO category)
```

Comment:
Renamed into ``addNoGoGoal``. Left the mandatoryness out of the name, as that's not part of the method behavior but part of the activity category that's passed to it.